idaes_examples/notebooks/docs/unit_models/custom_unit_models/creating_unit_model.ipynb

```diff
@@ -0,0 +1,2296 @@
+      "  10r 0.0000000e+00 3.81e+01 5.87e-09  -6.4 1.58e-01    -  1.00e+00 1.00e+00f  1\n",
+      "  11r 0.0000000e+00 3.91e+01 1.09e-05  -9.0 9.35e-01    -  9.68e-01 1.00e+00f  1\n",
+      "\n",
+      "Number of Iterations....: 11\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   5.1393961893966849e-07    5.1393961893966849e-07\n",
+      "Constraint violation....:   3.9105165554489545e+01    3.9105165554489545e+01\n",
+      "Complementarity.........:   9.0909090910996620e-10    9.0909090910996620e-10\n",
+      "Overall NLP error.......:   3.9105165554489545e+01    3.9105165554489545e+01\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 17\n",
+      "Number of objective gradient evaluations             = 5\n",
+      "Number of equality constraint evaluations            = 17\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 14\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 12\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.004\n",
+      "Total CPU secs in NLP function evaluations           =      0.000\n",
+      "\n",
+      "EXIT: Converged to a point of local infeasibility. Problem may be infeasible.\n",
+      "WARNING: Loading a SolverResults object with a warning status into\n",
+      "model.name=\"unknown\";\n",
+      "    - termination condition: infeasible\n",
+      "    - message from solver: Ipopt 3.13.2\\x3a Converged to a locally infeasible\n",
+      "      point. Problem may be infeasible.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'Problem': [{'Lower bound': -inf, 'Upper bound': inf, 'Number of objectives': 1, 'Number of constraints': 16, 'Number of variables': 16, 'Sense': 'unknown'}], 'Solver': [{'Status': 'warning', 'Message': 'Ipopt 3.13.2\\\\x3a Converged to a locally infeasible point. Problem may be infeasible.', 'Termination condition': 'infeasible', 'Id': 200, 'Error rc': 0, 'Time': 0.06552338600158691}], 'Solution': [OrderedDict([('number of solutions', 0), ('number of solutions displayed', 0)])]}"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "solver = pyo.SolverFactory(\"ipopt\")\n",
+    "solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7b17e30",
+   "metadata": {},
+   "source": [
+    "The model is infeasible thus indicating numerical issues with the model. We should call the `report_numerical_issues()` function and check what the constraints/variables causing this issue. "
```

Comment:
A minor technical note - this indicates the model is _probably_ infeasible, but does not guarantee it.